src/App.js

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+
+
+import PokemonCard from "./components/PokemonCard";
+
+import "./App.css";
 
 function App() {
+  const [pokemons, setPokemons] = useState(null);
+  const [pagina, setPagina] = useState(null);
+
+
+  useEffect(() => {
+    async function getPokemons() {
+      try {
+        const response = await axios.get(
+            `https://pokeapi.co/api/v2/pokemon/?offset=${pagina}&limit=20`
+        );
+        setPokemons(response.data.results);
+      } catch (error) {
+        alert("Oops, the pokemons got away");
+      }
+    }
+
+    getPokemons();
+  }, [pagina]);
+
+
   return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+  <>
+      <header>
+        <img className="logo" src="https://seeklogo.com/images/P/Pokemon-logo-497D61B223-seeklogo.com.png" alt="the word Pokemon"/>
+      </header>
+      <div className="pageButtons">
+          <button className="pageButton1" disabled={pagina === 0} onClick={() => setPagina(pagina - 20)}> Vorige </button>
```

Comment:
De Vorige knop is `disabled` als `pagina` `0` is. Alleen als de applicatie voor het eerste wordt opgestart, initialiseren we pagina met `null`. Terwijl we bij de eerste lading pokémons ook niet op "vorige" willen kunnen klikken, toch?